buildtools/webpack.ngeoexamples.js

```diff
@@ -2,23 +2,44 @@ const path = require('path');
 const ls = require('ls');
 const webpack = require('webpack');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
 
 const plugins = [];
 const entry = {};
 
 for (const filename of ls('examples/*.html')) {
   const name = filename.name;
-  entry[name] = ['ngeo/mainmodule.js', `./examples/${name}.js`];
+  entry[name] = [
+    './examples/common_dependencies.js',
+    'ngeo/mainmodule.js', // TODO check path!
+    `./examples/${name}.js`
+  ];
   plugins.push(
     new HtmlWebpackPlugin({
       template: `examples/${name}.html`,
       chunksSortMode: 'manual',
       filename: name + '.html',
       chunks: ['common', name],
-    }),
+    })
   );
 }
 
+// move data folder
+plugins.push(new CopyWebpackPlugin(
+  [
+    {
+      from: 'examples/data/*',
+      to: 'data',
+      flatten: true
```

Comment:
What is it for?

Reply:
Some examples use files from this folders. These must be copied to the compiled version for the examples to be usable.

Reply:
Yes, but I just speak about the flatten...

Reply:
Without the flatten it preserves the source folder structure.

Instead of `data/file.ext` I get `data/examples/data/file.ext`

There could be another combination of parameters to get the same result, but this seems to work.

Reply:
OK, thanks for your response :-)